.github/workflows/build.yml

```
@@ -120,6 +120,9 @@ jobs:
           IS_NIGHTLY: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/main') }}
           IS_RELEASE: ${{ startsWith(github.ref, 'refs/tags/') }}
           CAN_SIGN: ${{ secrets[matrix.config.certificate-secret] != '' }}
+          CREATE_USERNAME: ${{ secrets.CREATE_USERNAME }}
```

```suggestion
          # The CREATE_* environment vars are only used to run tests. These secrets are optional. Dependent tests will
          # be skipped if not available.
          CREATE_USERNAME: ${{ secrets.CREATE_USERNAME }}
```

I think some explanation can be useful for contributors who want to run the workflow in their fork.